docs/extended_api/concepts/group.md

```diff
@@ -0,0 +1,46 @@
+---
+grand_parent: Extended API
+parent: Concepts
+---
+
+# Group
+
+```c++
+struct Group {
+  static constexpr cuda::thread_scope thread_scope = /*implementation-defined*/;

```

Comment:
> I'm also unsure if this way of spelling it doesn't give the impression that the implementation defines what all Groups must define as the value here. Unsure what's a better way to spell it though.

I agree it might be confusing. We could just remove the `implementation-defined` comment and write it like so:
> `static constexpr cuda::thread_scope thread_scope;`

Thoughts?

Reply:
Yeah, I don't hate that.